tests/concurrent/test_concurrent.py

```diff
@@ -0,0 +1,33 @@
+import importlib
```

Comment:
It would be helpful if we could document the workflow and have few examples of usage.